llama_index/chat_engine/simple.py

```diff
@@ -65,6 +83,13 @@ async def achat(
 
         return Response(response=chat_response.message.content)
 
+    async def astream_chat(
```

Comment:
are we adding this later?

Reply:
Well, maybe lol this was a part I got stuck on.

Since the StreamingChatResponse actually uses an async generator, while the StreamingResponse does not, there would be a big UX and type difference between the simple and condense engine

A large part if this is that one is returning a query response, and the other an LLM response 😢

Reply:
Figured it out!